pkg/scheduler/testing/framework/fake_extender.go

```diff
@@ -137,6 +137,30 @@ func (pl *node2PrioritizerPlugin) ScoreExtensions() framework.ScoreExtensions {
 	return nil
 }
 
+type equalPrioritizerPlugin struct{}
```

Comment:
Can you use FakePreScoreAndScorePlugin instead of adding a new one?

Reply:
Changed NewEqualPrioritizerPlugin to be a wrapper creating a FakePreScoreAndScorePlugin.